src/pages/workspaces/workspace/analysis/modals/ComputeModal.js

```diff
@@ -638,7 +637,7 @@ export const ComputeModalBase = ({
       ..._.mapKeys(key => `desiredPersistentDisk_${key}`, desiredPersistentDisk),
       desiredPersistentDisk_costPerMonth: (desiredPersistentDisk && getPersistentDiskCostMonthly(getPendingDisk(), computeConfig.computeRegion)),
       ..._.mapKeys(key => `existingPersistentDisk_${key}`, existingPersistentDisk),
-      isDefaultConfig: !!simplifiedForm
+      isDefaultConfig: !!currentRuntimeDetails
```

Comment:
Hmm... is this correct? Don't see why this would change. With the original logic:

```
currentRuntimeDetails = undefined
simplifiedForm = true
isDefaultConfig = true
```

Now:
```
currentRuntimeDetails = undefined
isDefaultConfig = false
```

Reply:
so I should go with a triple exclamation? or some other logic? I'm not sure what to do in this case

Reply:
A single should suffice. Essentially, as I understand it, we are using the default config if no existing runtime details exist.

Reply:
thanks!